contracts/exchange-forwarder/contracts/src/MixinExchangeWrapper.sol

```diff
@@ -92,45 +92,90 @@ contract MixinExchangeWrapper is
         bytes[] memory signatures
     )
         internal
-        returns (FillResults memory totalFillResults)
+        returns (
```

Comment:
Since these functions no longer require `takerAssetData` to be WETH, let's rip out `weth` from all of our variable and function names. `_marketSellOrders` and `_marketBuyOrders` should be fine.

Reply:
There's still some contract logic that assumes the taker asset is WETH (e.g. keeping track of `wethSpentAmount`) so I think we can hold off on renaming this stuff until the next PR